ios/sdk/src/JitsiMeetConferenceOptions.m

```diff
@@ -71,6 +71,10 @@ - (void)setSubject:(NSString *_Nonnull)subject {
     [self setConfigOverride:@"subject" withValue:subject];
 }
 
+- (void)setDefaultRemoteDisplayName:(NSString *_Nonnull)defaultRemoteDisplayName {
```

Comment:
no need for the helper, setConfigOverride is a public API, so let's have them use it.